src/pages/offer/PeersTable.tsx

```diff
@@ -61,18 +69,26 @@ const PeerRow: React.FC<PeerRowProps> = ({ index, peer }) => {
             <Cell>
               <A href={`/peer/${peer.id}`}>{peer.id}</A>
             </Cell>
+            {/* Current CC */}
+            <Cell>
+              {peer.currentCapacityCommitment ? (
+                <A href={`/capacity/${peer.currentCapacityCommitment?.id}`}>
+                  {formatHexData(peer.currentCapacityCommitment?.id, 15, 15)}
+                </A>
+              ) : (
+                '-'
+              )}
+            </Cell>
             {/* Compute units */}
             <Cell>
               <Text size={12}>{peer.computeUnits.length}</Text>
             </Cell>
-            {/* Current Capacity Commitment */}
+            {/* Status */}
             <Cell>
-              {peer.currentCapacityCommitmentId ? (
-                <A href={`/capacity/${peer.currentCapacityCommitmentId}`}>
-                  {formatHexData(peer.currentCapacityCommitmentId, 20, 20)}
-                </A>
-              ) : (
-                <Text size={12}>-</Text>
+              {peer.currentCapacityCommitment?.status && (
+                <CapacityStatus
+                  status={peer.currentCapacityCommitment?.status}
+                />
```

Comment:
maybe there should be a `-` like above in case there is no peer.currentCapacityCommitment
or there should be no `-` above. Would be nice to be at least consistent with how missing values look in UI

Reply:
I have updated the code in this file to ensure consistent handling of missing values. Moving forward, I will apply this practice across other relevant sections to maintain uniformity